js/sites.js

```diff
@@ -265,20 +266,26 @@
      */
     refreshThumbnail() {
       gNewTab.sendToBrowser("NewTab:PageThumbs", {
-        url: this.url
+        link: this.link
       });
     },
 
-    _getURI(message) {
+    /**
+     * Render the correct thumbnail for the site.
+     *
+     * @param {Object} aData Contains enhanced links and the URI generated from the
+     *        current URL.
+     */
+    _getURI(aData) {
       // Only enhance tiles if that feature is turned on
-      let link = /*message.enhanced && DirectoryLinksProvider.getEnhancedLink(this.link) || */ this.link;
+      let link = aData.enhanced && this.link;
```

Comment:
Ok, I see. That's confusing tho :( Can we put this into a ternary or an if statement?